index.js

```diff
@@ -57,6 +60,22 @@ client.on('message', (message) => {
     message.reply(message.author.displayAvatarURL())
     ev.addField('avatar', true)
   }
+
+  if (message.content === '!Announcement' && message.channel.id === '953362393071775814'){
+    await message.author.send('What is your announcement?')
+
+    const filter = (m) => m.author.id === message.author.id
+    const userAnnouncement = await message.author.dmChannel.awaitMessages(filter, {max: 1})
+      .then((collected) => collected.first().content)
+    if (userAnnouncement === 'cancel'){
+      await message.author.send('Canceled the announcement.')
+      message.delete()
```

Comment:
If you decide that you want to cancel the announcement, just type `cancel` in the dm message and the bot will cancel it and let you know that the announcement was canceled.